opentelemetry/src/trace/span.rs

```diff
@@ -260,4 +262,14 @@ impl StatusCode {
             StatusCode::Error => "ERROR",
         }
     }
+
+    /// Return the priority of the status code.
+    /// Status code with higher priority can override the lower priority one.
+    pub fn priority(&self) -> i32 {
```

Comment:
Could this method be private?

Reply:
Probably not private, but we can make it public within the crate.